### PR TITLE
Update carbon chain info

### PR DESCRIPTION
**Pull request by stevenkhong**
- add currencies
- update gas price steps

### CODE REVIEW

File: cosmos/carbon.json [hunk 1]
```diff
@@ -71,5 +251,9 @@
     "coinDecimals": 8,
     "coinGeckoId": "switcheo"
   },
-  "features": []
-}
\ No newline at end of file
+  "features": [
```
Comment (HeesungB) [inline cosmos/carbon.json:254]: features: any other features that are additionally supported by the chain
cosmwasm: supports CosmWasm smart contracts
secretwasm: supports WASM smart contracts of Secret Network
eth-address-gen: supports EVM account generation
eth-key-sign: supports EVM signatures
axelar-evm-bridge: supports EVM bridge provided by Axelar Network
osmosis-txfees: supports paying fees in other currencies on Osmosis

so Could you remove "ibc-transfer", "ibc-go" field?

Comment (stevenkhong) [reply cosmos/carbon.json:254]: thanks, corrected

**Comment by stevenkhong** [comment]
Carbon uses non-default gas costs ([recorded here](https://api.carbon.network/carbon/fee/v1/gas_costs)), and the seemingly random gas prices is a workaround calculated to match these gas costs when assuming a gas cost of 130000 (default MsgSend cost). 

e.g. Carbon gas cost for `/cosmos.bank.v1beta1.MsgSend` is 100,000,000, while default used on Keplr is 130,000. 
Carbon has min gas price of 1 swth, which means the gas fee will be 100,000,000 x 1 = 100,000,000.
While Keplr using 130,000 gas and gas price provided in this PR, will produce the gas fee 130,000 x 769.2307692307693 = 100,000,000.

**Comment by stevenkhong** [comment]
@HeesungB sorry for the bump, does everything look alright with this PR? We need this for Keplr to operate properly on carbon chain.

**Comment by HeesungB** [comment]
@stevenkhong Thank you for contribution!

**Pull request merged**